src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -63,6 +63,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.7.4" />
```

Comment:
small nit: I think the reason we had two separate `<ItemGroup>` sections was because one has all of the private assets (i.e. only used for dev purpose) and the other has our actual dependencies. I think we should move these dependencies back to the other `<ItemGroup>` exception with the rest of our real dependencies.